src/test/unit/math/SparseBinaryMatrixTest.cpp

```diff
@@ -30,6 +30,7 @@
 #include <gtest/gtest.h>
 
 #include <nupic/math/SparseBinaryMatrix.hpp>
+using namespace capnp;
```

Comment:
@rcrowder How about move `using namespace` directives into between the `includes` and the class. Something like this:

```
#include <nupic/math/SparseBinaryMatrix.hpp>
#include <nupic/proto/SparseBinaryMatrixProto.capnp.h>
#include <nupic/types/Types.h>

using namespace capnp;

namespace nupic
{
```